utils/install-app.js

```diff
@@ -139,6 +141,88 @@ function addToServer(appDir, installDir) {
   }
 }
 
+function loadRecognizers(appDir, appId, appVers) {
+  let recognizers;
+  let configRecognizers;
+  let configLocation;
+
+  try {
+    recognizers = JSON.parse(fs.readFileSync(path.join(appDir, "config/recognizers", appId))).recognizers;
+    const keys = Object.keys(recognizers)
+    for (const key of keys) {
+      recognizers[key].pluginVersion = appVers;
+      recognizers[key].pluginIdentifier = appId;
+    }
+    logger.debug('ZWED0294I', appId); //logger.debug("Found recognizers for '" + appId + "'");
+  } catch (e) {
+    logger.debug('ZWED0295I', (path.join(appDir, "config/recognizers", appId)), appId); //logger.debug("Could not find recognizers in '" + (path.join(appDir, "config/recognizers", appId)) + "'");
```

Comment:
Here we get config & recognizers for the ones existing in a plugin (/appDir/config/xxx)